CondTools/L1TriggerExt/plugins/L1CondDBPayloadWriterExt.cc

```diff
@@ -66,6 +78,8 @@ void L1CondDBPayloadWriterExt::analyze(const edm::Event& iEvent, const edm::Even
 
   // Get L1TriggerKeyListExt and make a copy
   L1TriggerKeyListExt oldKeyList;
+  l1t::DataWriterExt& m_writer = *m_rcdToWriterMap["L1TriggerKeyExtRcd@L1TriggerKeyExt"];
```

Comment:
sure, makes sense - I will apply the change

(actually this key should always be present, as the query based on this key provides the list of the other keys that we need to loop over - so even if this will be the only call to the writer it should always be present)